forest/cli/src/cli/snapshot_cmd.rs

```diff
@@ -323,6 +349,128 @@ fn clean(config: &Config, snapshot_dir: &Option<PathBuf>, force: bool) -> anyhow
     Ok(())
 }
 
+async fn validate(
+    config: &Config,
+    recent_stateroots: &i64,
+    snapshot: &PathBuf,
+) -> anyhow::Result<()> {
+    let confirm = Confirm::with_theme(&ColorfulTheme::default())
```

Comment:
We should only prompt if stdin is a tty. And we also need some kind of flag to force the validation.